src/Model.elm

```diff
@@ -241,43 +246,39 @@ decodeRecord =
         (field "last_modified" int)
 
 
-sendFormData : Form.Model -> Cmd Msg
-sendFormData formData =
-    -- TODO: handle auth with provided credentials
+sendFormData : Model -> Form.Model -> Cmd Msg
+sendFormData model formData =
     let
-        rootUrl =
-            "https://kinto.dev.mozaws.net/v1/buckets/default/collections/test-items/records"
-
-        ( method, url, responseMsg ) =
-            case formData.id of
-                Nothing ->
-                    ( HttpBuilder.post, rootUrl, CreateRecordResponse )
-
-                Just id ->
-                    ( HttpBuilder.patch, rootUrl ++ "/" ++ id, EditRecordResponse )
-
-        request =
-            method url
-                |> withHeader "Authorization" "Basic dGVzdDp0ZXN0"
-                |> withJsonBody (encodeFormData formData)
-                |> withExpect (Http.expectJson (field "data" decodeRecord))
+        data =
+            encodeFormData formData
     in
-        HttpBuilder.send responseMsg request
-
-
-deleteRecord : RecordId -> Cmd Msg
-deleteRecord recordId =
-    -- TODO: handle auth with provided credentials
-    let
-        delete_url =
-            "https://kinto.dev.mozaws.net/v1/buckets/default/collections/test-items/records/" ++ recordId
-
-        request =
-            HttpBuilder.delete delete_url
-                |> withHeader "Authorization" "Basic dGVzdDp0ZXN0"
-                |> withExpect (Http.expectJson (field "data" decodeRecord))
-    in
-        HttpBuilder.send DeleteRecordResponse request
+        case formData.id of
+            Nothing ->
+                Kinto.createRecord
+                    model.kintoConfig
+                    "default"
+                    "test-items"
+                    data
+                    CreateRecordResponse
+
+            Just recordId ->
+                Kinto.updateRecord
+                    model.kintoConfig
+                    "default"
+                    "test-items"
+                    recordId
+                    data
+                    EditRecordResponse
```

Comment:
Not sure if it's a common Elm idiom, or if it could/should be improved, will revisit this later, thanks!